composer.json

```diff
@@ -30,12 +30,10 @@
         "phpunit/PHPUnit": "~4.0"
     },
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
-        "zendframework/zend-db": "Zend\\Db component",
+        "ext-mongo": "mongodb extetension to use MongoDB writer",
         "zendframework/zend-escaper": "Zend\\Escaper component, for use in the XML formatter",
-        "zendframework/zend-mail": "Zend\\Mail component",
-        "zendframework/zend-validator": "Zend\\Validator component"
+        "zendframework/zend-mail": "Zend\\Mail component to use Email writer",
+        "zendframework/zend-validator": "Zend\\Validator component to use filter feature"
```

Comment:
Ok, I see. It does however do more than just filtering. [It blocks messages](https://github.com/zendframework/zend-log/blob/be6598c6e6dfa93368850b89e2d26fc4236cb29f/src/Filter/Validator.php#L50) from being logged if they do not pass validation. In that case I think the description can be a bit more explanatory like: `Zend\\Validator component to block invalid log messages`